.vscode/tasks.json

```diff
@@ -11,6 +11,34 @@
                 "cwd": "${workspaceRoot}"
             },
             "problemMatcher": [],
+        },
+        {
+            "type": "shell",
+            "label": "kubectl port-forward",
+            "command": "kubectl",
+            "args": [
+                "port-forward",
+                "-n",
+                "${input:namespace}",
+                "${input:pod}",
+                "5678",
```

Comment:
I think this has to be "5678:5678" or it just uses some rando port and then you have to change your launch config to the use the port it chooses. Presumably we could then add "preLaunchTask": "kubectl port-forward" back into the launch .json as it is still necessary to do the port forward manually in a terminal to get it to work at the moment.